# 追击指令消息
bool chase_active       # 是否激活追击模式的标志
geometry_msgs/Point relative_position # 敌人的相对位置 (x, y)